auth_oidc/readme/CONFIGURE.rst

```diff
@@ -16,14 +14,24 @@ to the client, and in logs.
   <https://docs.microsoft.com/en-us/powerapps/maker/portals/configure/configure-openid-settings>`_
   for more information):
 
-* Provider Name: Azure
-* Auth Flow: OpenID Connect
-* Client ID: use the value of the OAuth2 autorization endoing (v2) from the Azure Endpoints list
-* Body: Azure SSO
-* Authentication URL: use the value of "OAuth2 autorization endpoint (v2)" from the Azure endpoints list
-* Scope: openid email
-* Validation URL: use the value of "OAuth2 token endpoint (v2)" from the Azure endpoints list
+.. image:: ..static/description/oauth-microsoft_azure-api_permissions.png
+
+.. image:: ..static/description/oauth-microsoft_azure-optional_claims.png
+
+* Provider Name: Azure AD Single Tenant
```

Comment:
What's the differences between auth with single or multi ? pros/cons ?

Reply:
Is this sentence sufficient? Or should I add this link as well? 
https://learn.microsoft.com/en-us/azure/active-directory/develop/single-and-multi-tenant-apps

Reply:
it's fine